api/src/services/flows.ts

```diff
@@ -9,48 +9,30 @@ export class FlowsService extends ItemsService<FlowRaw> {
 	}
 
 	override async createOne(data: Partial<Item>, opts?: MutationOptions): Promise<PrimaryKey> {
-		const flowManager = getFlowManager();
-
 		const result = await super.createOne(data, opts);
-		await flowManager.reload();
-
-		return result;
-	}
-
-	override async createMany(data: Partial<Item>[], opts?: MutationOptions): Promise<PrimaryKey[]> {
-		const flowManager = getFlowManager();
-
-		const result = await super.createMany(data, opts);
-		await flowManager.reload();
-
-		return result;
-	}
 
-	override async updateBatch(data: Partial<Item>[], opts?: MutationOptions): Promise<PrimaryKey[]> {
 		const flowManager = getFlowManager();
-
-		const result = await super.updateBatch(data, opts);
 		await flowManager.reload();
```

Comment:
I'd don't know the overhead of `flowManager.reload()`, can you tell me the performance implications? Cause in this cause it might make sense to specifically create a generic `ItemsService` on which to call the `create`/`update` methods so it only gets reloaded once.